suite-common/wallet-core/src/transactions/transactionReducer.ts

```diff
@@ -0,0 +1,145 @@
+import { accountsActions } from 'suite-common/wallet-core/src/index';
```

Comment:
please rename this file to transaction**s**Reducer.ts (you missing that "s" at the end of transaction)